samples/RenderTest/SideBar.xaml

```diff
@@ -1,11 +1,11 @@
-<Styles xmlns="https://github.com/avaloniaui">
+<Styles xmlns="https://github.com/avaloniaui" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">>
```

Comment:
There's an extra `>` here that's causing this XAML to fail on load.

Reply:
Fixed!